WebContent/WEB-INF/jsp/eventHandlers.jsp

```diff
@@ -508,6 +508,7 @@
 
             selectedHandlerNode.object = handler;
         }
+        $set("xid", handler.xid);
```

Comment:
Where is the definition of the handler variable?
What is defined in the else block is not available outside this block.
If the xid has not been completed when editing the handler, it should not save the newly generated xid and return the old xid. Otherwise, this is counterintuitive behavior and the user may unknowingly change the xid of the handler.